src/openms/CMakeLists.txt

```diff
@@ -153,6 +154,7 @@ openms_add_library(TARGET_NAME  OpenMS
                    PRIVATE_INCLUDES ${EOL_BSPLINE_INCLUDE_DIRECTORY} ${ISOSPEC_INCLUDE_DIRECTORY} 
                    EXTERNAL_INCLUDES ${OPENMS_CONTRIB_INCLUDE_DIRS}
                                      ${OpenSwathAlgo_INCLUDE_DIRECTORIES}
+                                     ${EVERGREEN_INCLUDE_DIRECTORY}
```

Comment:
is there a corresponding commit in the contrib?

Reply:
It actually is currently used as a header-only library here in the repository, like IsoSpec.

Reply:
ok, sounds good. Please make sure you put it into its own namespace (we had to do the same thing for IsoSpec). Otherwise we add functions to the global namespace through inclusion of OpenMS headers which can have unwanted side effects